std_msgs/Header header
int64 depthFromUSBL             # depthFromUSBL reading
int64 depthFromUSBL_variance    # depthFromUSBL_variance reading

